docs/src/main/sphinx/connector/hive-azure.rst

```diff
@@ -115,6 +115,12 @@ catalog configuration.
     - Value of the new client (application) secret created
   * - ``hive.azure.adl-refresh-url``
     - OAuth 2.0 token endpoint url
+  * - ``hive.azure.adl-proxy-host``
+    - Proxy host and port in ``host:port`` format. Use this property to connect
+      to an ADLS endpoint via an internet proxy from a host from within an
```

Comment:
The last part "from a host from ..." makes no sense. This is for Trino connecting to ADLS, unrelated to HMS.